project/main.py

```diff
@@ -61,6 +62,8 @@ def main():
     rootLogger.addHandler(fileHandler)
     rootLogger.addHandler(consoleHandler)
 
+    configMap = readConfigFile(args.config,logging)
```

Comment:
Shouldn't need to pass logging here

Reply:
same?

Reply:
This is the only place that readConfigFile is called.